client/src/components/existing-request/existing-request-display.vue

```diff
@@ -528,6 +530,9 @@ export default class ExistingRequestDisplay extends Mixins(
         case NrAction.RETRY_PAYMENT:
           this.navigateToPaymentPortal()
           break
+        case NrAction.RESULTS:
```

Comment:
Does this button have a loading indicator while the results are being created/downloaded?

Reply:
I utilized the global spinner opposed to locking out the individual button. I figured since there was multiple actions a user could click on, the global spinner was a good choice. Any thoughts on this?

Reply:
Ah, that works.

I have no preference, other than having a common mechanism -- but which is the common mechanism in NR?

TLDR - you don't need to change this (unless, say, Scott/Tracey have a preference).

Reply:
PS I think the UI would support downloading the results AND doing something else concurrently (unless there are global state conflicts). Again, I have no pref whether to block just this button or the whole UI.

PPS What does the error handling look like?

Reply:
Oh, interesting, right!
Currently were logging the error but i will add something in for the UI.
ie a modal, although i didn't see a design for that. Possibly use a generic confirm, as a failure would/could just be tried again with no detriment.

Reply:
A simple error modal would be OK. The user can simply retry from the main page.

Reply:
(The error modal may already display.)

Reply:
Added! Thanks, good catch 😁